packages/ai/manager.py

```
@@ -0,0 +1,106 @@
+        ordered_providers = list(self.providers)
+        if model:
+            try:
+                from packages.ai.registry import get
+                info = get(model)
+                if info and info.provider_id:
+                    owning = [p for p in ordered_providers if p.provider_id == info.provider_id]
+                    if owning:
+                        # Move owning provider to front, keep others as fallback.
+                        ordered_providers = owning + [p for p in ordered_providers if p.provider_id != info.provider_id]
+            except Exception:  # noqa: BLE001 — registry lookup is best-effort
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)